### PR TITLE
Feature/12 map selector

### PR DESCRIPTION
**Pull request by Eirmas**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,11 @@
 NODE_ENV=development
+<<<<<<< HEAD
```
Comment (NikoGrano) [inline .env:2]: @Eirmas Never commit this file, even if it is draft

File: .env-example [hunk 2]
```diff
@@ -1,5 +1,5 @@
 NODE_ENV=
-VUE_APP_MS_SOCK=
+VUE_APP_MS_SOCK=wsya://socket.gametactic.eu
```
Comment (NikoGrano) [inline .env-example:2]: Revert this line

File: package.json [hunk 3]
```diff
@@ -16,6 +16,7 @@
     "core-js": "^3.6.4",
     "jsonwebtoken": "^8.5.1",
     "konva": "^4.0.18",
+    "lodash": "^4.17.15",
```
Comment (NikoGrano) [inline package.json:19]: @bryansamuel1 required?
@Eirmas Comment?

File: .env (hunk 1 above)
Comment (bryansamuel1) [reply .env:2]: Yeah these are conflict lines, wtf are they doing here? Hah

File: package.json (hunk 3 above)
Comment (bryansamuel1) [reply package.json:19]: not the whole library no. Please only add the packages that contain specific lodash functions. This for example is the one for https://www.npmjs.com/package/lodash.range lodash range function

File: src/components/TheCanvas.vue [hunk 4]
```diff
@@ -18,15 +18,15 @@
 <script lang="ts">
 import { Prop } from 'vue-property-decorator'
 import Component from 'vue-class-component'
-import { Tool, Tracker } from '@/tools/Tool'
+import { ToolInterface, Tracker } from '@/tools/Tool'
```
Comment (bryansamuel1) [inline src/components/TheCanvas.vue:21]: I don't like naming an interface as interface. Is there a different name you could use?

File: src/components/canvas-tools/templates/template-tools/ColourPicker.vue [hunk 5]
```diff
@@ -27,7 +27,7 @@ export default class ColourPicker extends Vue {
     ['#D74E09FF', '#792387FF'],
     ['#F2BB05FF', '#3349F4FF'],
     ['#08A045FF', '#4392F1FF'],
-    ['#F7FFF7FF', '#080F0FFF']
```
Comment (bryansamuel1) [inline src/components/canvas-tools/templates/template-tools/ColourPicker.vue:30]: why has this colour changed?

File: src/components/entity-panel/sections/TheCreateEntity.vue [hunk 6]
```diff
@@ -181,6 +181,11 @@
             />
           </v-col>
         </v-row>
+        <v-subheader>
+          <p class="caption">
+            <b>Note</b> Drag and drop cards onto the canvas to create them
```
Comment (bryansamuel1) [inline src/components/entity-panel/sections/TheCreateEntity.vue:186]: please translate

File: src/components/overlays/TheCreateNewTacticOverlay.vue [hunk 7]
```diff
@@ -0,0 +1,126 @@
+<template>
+    <v-dialog persistent v-model="overlay" width="700" class="custom-overlay">
```
Comment (bryansamuel1) [inline src/components/overlays/TheCreateNewTacticOverlay.vue:2]: please implement translations in this file

File: src/tools/Line.ts [hunk 8]
```diff
@@ -46,25 +46,24 @@ export default class Line implements LineInterface {
     canvasElement.position = this.lineCreator.getGroup().position()
   }
 
-  // eslint-disable-next-line
-  mouseMoveAction = throttle((event: CustomEvent, canvasElement: CanvasElement, layer: Konva.Layer, socket: WebSocket): void => {
+  mouseMoveAction = (event: CustomEvent, canvasElement: CanvasElement, layer: Konva.Layer): void => {
```
Comment (bryansamuel1) [inline src/tools/Line.ts:49]: you are removing the throttle?

File: src/tools/util/MapCanvas.ts [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  createMapElement = (stageConfig: CustomStageConfig, layer: Konva.Layer): void => {
+    const mapDimentionRatio = 0.75
+    const imgObj = new Image()
+    imgObj.onload = () => {
+      const el = new Konva.Image({
+        image: imgObj,
+        x: (stageConfig.initialWidth * ((1 - mapDimentionRatio) / 2)),
+        y: (stageConfig.initialHeight * ((1 - mapDimentionRatio) / 2)),
```
Comment (bryansamuel1) [inline src/tools/util/MapCanvas.ts:21]: Dimension not Dimention*

File: src/types/Games/Wows/Maps.ts [hunk 10]
```diff
@@ -0,0 +1,279 @@
+import { Map } from '@/store/modules/types'
+
+export default class WowsMaps {
+  private maps: Map[] = [
+    {
+      desc: 'Somewhere in the Atlantic.',
```
Comment (bryansamuel1) [inline src/types/Games/Wows/Maps.ts:6]: These maps will need translations if the user can see this english in the client

File: src/views/Index.vue [hunk 11]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <p>You're begin redirected...</p>
+  <p>You're being redirected...</p>
```
Comment (bryansamuel1) [inline src/views/Index.vue:2]: add translations

**Pull request closed without merging**